app/assets/javascripts/modules/routes.js

```diff
@@ -150,6 +155,16 @@ angular.module('probedock.routes', [ 'ui.router' ])
         url: '/:id'
       })
 
+      .state('org.project', {
+        abstract: true,
+        template: '<div ui-view />'
+      })
+
+      .state('org.project.show', {
```

Comment:
For consistency with `org.users.show`, use `org.projects.show` (and thus remove the duplicate previous abstract state).